mappings/net/minecraft/entity/ai/pathing/BirdPathNodeMaker.mapping

```diff
@@ -1 +1,7 @@
 CLASS net/minecraft/class_6 net/minecraft/entity/ai/pathing/BirdPathNodeMaker
```

Comment:
Does this still only apply to birds (parrots), or does it apply to bees as well?